install.sh

```diff
@@ -76,10 +78,15 @@ else
 	echo "cURL and wget not found! please install one of them"
 	exit 1
 fi
+
+system "rm /usr/local/bin/ssc"
 system "sudo chmod 755 /usr/local/bin/ssc"
-system "sudo ln -s /usr/local/bin/ssc /usr/local/bin/sshconfig"
+system "sudo ln -s /usr/local/bin/ssc /usr/local/bin/ssc"
```

Comment:
You are linking ssc to itself  , in early versions of ssc we use sshconfig as our main command existing of this link is because of that .